src/concrete/ml/sklearn/base.py

```diff
@@ -1955,7 +1962,7 @@ def gather1d(x, indices):
                 arr = []
                 for i in indices:
                     arr.append(x[i])
-                enc_arr = fhe_array(arr)
+                enc_arr = cnp.array(arr)
```

Comment:
don't do it here I think but we'll remove this `cnp` mention later : https://github.com/zama-ai/concrete-ml-internal/issues/4074

Reply:
ok

Reply:
what I mean is that `cnp` is a reference to concrete-numpy which we don't use anymore

Reply:
I understood. 
I mean, I prefer to keep this naming, since we are planning to remove them in https://github.com/zama-ai/concrete-ml-internal/issues/4074.

Tracking cnp.X is easier than tracking fhe_array, fhe_zeros + cnp.X.  Don't you think?

Reply:
yes I agree that's what I said "don't do it here I think" 😅  I meant "we should update this cnp naming but later, it was just a comment saying that I've just realized this in your PR, which made me created the above PR !